go.mod

```diff
@@ -28,4 +32,4 @@ require (
 // has go module support, we'll need this exclude, and perhaps some more.
 //
 // After that point, we should be able to remove it.
-exclude gopkg.in/alecthomas/kingpin.v3-unstable v3.0.0-20180810215634-df19058c872c
```

Comment:
I'm not sure if this should be removed or not. Ideally we could use [`golangci-lint`](https://github.com/golangci/golangci-lint) instead of gometalinter

Reply:
That is actually coming from `go mod tidy`

Reply:
I will see how build goes and take action upon it.